client/concordclient/include/client/concordclient/event_update_queue.hpp

```diff
@@ -73,6 +73,8 @@ class UpdateQueue {
   virtual std::unique_ptr<EventVariant> tryPop() = 0;
 
   virtual uint64_t size() = 0;
+
+  virtual void setException(std::exception_ptr e) = 0;
```

Comment:
There is probably a transient include happening. We should add `#include <exception>`.

Reply:
Updated.